views/js/previewer/plugins/tools/scale/scale-wrapper.tpl

```diff
@@ -0,0 +1,13 @@
+<div class="preview-overlay tao-scope overlay preview-{{type}} item-no-print">
+    <div class="preview-canvas">
+        <div class="preview-scale-container">
+            <div class="{{type}}-preview-frame preview-outer-frame {{type}}-preview-landscape">
+                <div class="{{type}}-preview-container preview-container">
```

Comment:
This seems too complex. Are those classes actually needed? Why not simply use a single place to `type` and then rely on inheritance to apply styles?
More places you add, more difficult it is to manage.

```html
<div class="tao-scope preview-overlay item-no-print" data-type="{{type}}"  data-mode="{{orientation}}">
    <div class="preview-canvas">
        <div class="preview-scale-container">
            <div class="preview-frame">
                <div class="preview-container">
```